app/helpers/tag_lists_helper.rb

```diff
@@ -15,7 +10,20 @@ def available_tags_checkboxes(tags)
     end, '')
   end
 
-  private
```

Comment:
Restore `private` in an appropriate place, so that the methods below your new ones don't get exposed

Reply:
well, methods inside helpers are exposed anyway. it's more informative to have it declared private. Going to restore it